packages/client/src/client/notifications.openfin.ts

```diff
@@ -335,15 +388,49 @@ export const registerCreditAcceptedNotifications = (
       handler || handleHighlightCreditBlotterAction,
     )
 
-    acceptedRfqWithQuote$.subscribe((rfq) => {
-      sendQuoteAcceptedNotification(rfq)
-    })
+    acceptedRfqWithQuoteSubscription = acceptedRfqWithQuote$.subscribe(
+      (rfq) => {
+        sendQuoteAcceptedNotification(rfq)
+      },
+    )
+  }
+}
+
+let areCreatedCreditNotificationsRegistered = false
```

Comment:
minor: should not need individual guard flags for the three Credit registrations